### ISSUE TITLE
Mismatch in Progressive Self-Evolution Regularization: Code Implementation Differs from the Paper

### ISSUE THREAD

**Issue by Wei-Nijuan** [aopolin-lv/DecisionMamba#1]
I found the code for computing Progressive Self-Evolution Regularization Loss in the seq_trainer:
```python
soft_action_target = ((1 - alpha_t) * action_target) + (alpha_t * action_preds)
loss_a = torch.mean((action_preds - soft_action_target) ** 2)
```
However, in the calculation of soft_action_target, why does it use action_preds generated by the current policy instead of those generated by the previous policy?

**Comment by aopolin-lv** [comment]
sry, let me check.

**Comment by aopolin-lv** [comment]
Sorry for the lack of proper code version management, which has led to issues with the `seq_trainer.py` code. We have retrieved all codes from that time and are currently rerunning them to verify the accuracy of the previous experiments. This process will take some time, and we appreciate your patience.

**Comment by Drama1234** [comment]
It's been months. Can you solve this problem?
